src/backends/perf_pt/perf_pt.c

```diff
@@ -400,14 +406,20 @@ perf_pt_init_tracer(struct tracer_conf *tr_conf)
 /*
  * Turn on Intel PT.
  *
+ * `trace_bufsize` is the starting capacity of the trace buffer.
+ *
+ * The trace is written into `*trace_buf` which may be realloc(3)d. The trace
+ * length is written into `*trace_len`.
+ *
  * Returns zero on success, or -1 otherwise.
  */
 int
-perf_pt_start_tracer(struct tracer_ctx *tr_ctx)
+perf_pt_start_tracer(struct tracer_ctx *tr_ctx, struct perf_pt_trace *trace)
 {
     int clean_sem = 0, clean_thread = 0;
     int ret = 0;
 
+
```

Comment:
Double blank line?

Reply:
Oops, yes. That'll be a debugging leftover.